docs/pages/eas-update/runtime-versions.mdx

```diff
@@ -100,7 +100,27 @@ It's important to know that this policy does require the developer to manage the
 
 Also, if you select a different native version between Android and iOS, you'll end up with builds and updates with separate runtime versions. When you publish an update, EAS CLI will detect that two updates are needed, and it will create two updates with the required runtime versions and publish them separately.
 
-### `"fingerprintExperimental"` runtime version policy (experimental)
+### `"fingerprint"` runtime version policy
```

Comment:
Why not just leave it as `fingerprintExperimental`? Is it so people who were relying on it in sdk 50 and below will need to make an active decision to switch?

Reply:
It's a valid question. Your guess is correct. They're incompatible with each other, so we don't want people who used `fingerprintExperimental` to accidentally be upgraded to `fingerprint` automatically.